windows/translate/nl.nsi

```diff
@@ -0,0 +1,37 @@
+; Dutch translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_DUTCH} \
+    "Desktop shortcut maken"
+
+LangString INVALID_FOLDER_MSG ${LANG_DUTCH} \
+    "De doelmap bestaat al. Kies een nieuwe doelmap."
+
+LangString RUNNING_APP_MSG ${LANG_DUTCH} \
+    "${APP_NAME} loopt nog. Sluit het programma en start de installatie opnieuw.."
+
+LangString OLD_WRONG_VER_FOUND ${LANG_DUTCH} \
+    "We hebben een oudere versie van ${APP_NAME} in de 32 Bit Program Files map gevonden. Deze kan het beste worden verwijderd alvorens een nieuwe versie van ${APP_NAME} te installeren. Wilt u het nu laten verwijderen?"
+
+LangString OLD_WRONG_VER_FOUND_CONFIRM ${LANG_DUTCH} \
+    "Als u doorgaat zonder het te verwijderen, kan de installatie mogelijk mislukken! Weet u zeker dat u de oude versie niet wil verwijderen?"
+
+LangString OLD_VER_REMOVE_FAILED ${LANG_DUTCH} \
+     "FATALE FOUT: HET DE-INSTALLATIEPROGRAMMA HEEFT NIET GEWERKT. Zodra u op OK klikt zal de oude versie op uw PC blijven en proberen we de nieuwe versie te installeren. U kunt ook op annuleren klikken en het zelf proberen te verwijderen."
```

Comment:
I think we should re-think this in english. At least my german background tells me that this sentence might need to be rephrased. @gilgongo

Reply:
I guess we could soften it a bit? So: 

"Sorry, we couldn't uninstall the old version. We'll try to install the new one, but you can also press cancel and try to remove the old version on your own."

Reply:
Yes, Dutch is generally quite "direct". I will soften it a bit whilst staying as close to the English as possible.

Reply:
Agree. We should change this to gilgongo's proposal.

Reply:
@jerogee I think we should change it to gilgongo‘s proposal

Reply:
Done